src/Build.UnitTests/Evaluation/Evaluator_Tests.cs

```diff
@@ -4829,23 +4829,23 @@ public void VerifyPropertyTrackingLoggingPropertyInitialValue()
 
                     // Verify logging of property initial values.
                     propertyInitialValueMap.ShouldContainKey("Prop");
-                    propertyInitialValueMap["Prop"].PropertySource.ShouldBe("Xml");
```

Comment:
still validate the source? or does it not make sense now?

Reply:
I don't pass the source if File path is known. It seems to be excessive now.